### PR TITLE
 add 3 histgrams of hScaling in AliAnalysisTaskEmcalJetPerformance.cxx

### PR DESCRIPTION
**Pull request by 11kumaoka**
I want to add 3histgrams fo hScaling in AliAnalysisTaskEmcalJetPerformance.cxx, to check the dependence of the scal value on centrality.

### CODE REVIEW

**Comment by 11kumaoka** [comment]
Add a new task for output raw jet pt spectrum divided by 10% centrality. @jdmulligan

**Comment by ktf** [comment]
+test

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# ALICE
```
Comment (raymondEhlers) [inline README.md:1]: Why is this being edited?

**Comment by 11kumaoka** [comment]
Dear Raymond,

I want to know why the scale value depends on centrality.
Therefore, I want to check the behavior of each parameter which makes up the scale value.

Best regards,
Takuya

###########################################################
1-1-1 Tennodai, Tsukuba, Ibaraki, 305-8571 Japan.
Faculty of Pure and Applied Sciences, Division of Physics, Expermental Nuclear Physics
Tsukuba University

Takuya Kumaoka
2020年8月11日 1:42 +0900、Raymond Ehlers <notifications@github.com> のメール:

**Comment by raymondEhlers** [comment]
Dear Takuya,

Please look more closely at where I commented - it wasn't on your analysis task. You added a new `README.md` at the root of the repository with the contents of:

```
# ALICE
# ALICE
# ALICE
```

It's unrelated to your PR, and I assume that you didn't mean to add it in. Please remove it, and then I can approve your PR.

Looking a bit more closely at your actual PR: You'll need to add your new analysis task to the CMakeLists.txt and the LinkDef.

Best,
Raymond

**Comment by 11kumaoka** [comment]
Thank you for giving me advice.
However, I could not understand the mean of "# ALICE".
You mean I should remove the new task "AliAnalysisTaskRawJetSpec", don't you?

**Comment by raymondEhlers** [comment]
No, that's not what I mean. You can see the files that you've changed in the PR here: https://github.com/alisw/AliPhysics/pull/15173/files . With your 9 commits, you only have changes to the JetPerformanceTask and added an unrelated `README.md`.  As I said above, I'm talking about the `README.md` file that you I assume you accidentally added in the root of the repository. 

I think you need some help with git at this point. That's okay - we all went through a learning curve with it! Unfortunately, I don't have the time to walk you through this right now. I'm not sure who you're working with (perhaps @jdmulligan ?), but I suggest that you ask that person or someone at your institute with git knowledge to meet with you to help sort out your pull request

**Comment by 11kumaoka** [comment]
Thank you for quickly replying!
I will ask a person with git knowledge.

**Comment by raymondEhlers** [comment]
Very good - you removed the README.md, so your making progress! Two issues remain:

- I think you decided to add a new task rather than modify the JetPerformanceTask. Right now, the PR doesn't contain the new task. It just has the changes to the existing task. So please keep working on this
- There are now a ton of commits in this PR. Once the above point is resolved, please squash them down to one commit.

You might need to consult with your local git expert again

**Comment by 11kumaoka** [comment]
Thank you for giving me advice again!
I will ask my git expert again.

**Pull request closed without merging**